### ISSUE TITLE
Not working on Brave Beta/Nightly && Chrome Canary

### ISSUE THREAD

**Issue by PleahMaCaka** [anaclumos/bing-chat-for-all-browsers#104]
[ ✔️ ] Chrome 113.0.5672.127
 [ ✔️ ] Chrome Beta 114.0.5735.45
 [ ❌ ] Chrome Canary 116.0.5794.0 - first time once not working
 [ ✔️ ] Brave Latest 1.51.118 (Chromium 113.0.5672.126) 
 [ ❌ ] Beta Beta 1.53.66 (Chromium 114.0.5735.35)
 [ ❌ ] Brave Nightly 1.54.9 (Chromium 114.0.5735.53)
Once after installation this extension works, but when I close the tab and try again, it doesn't work in Brave Beta & Nightly.

**Comment by cy18** [comment]
"Once after installation this extension works, but when I close the tab and try again, it doesn't work in Brave Beta"

The similar problem occurs to me. I just switched from Vivaldi 6.0.2979.18  to Brave 1.53.67 today. When the extension is just installed, Bing chat works. Then it stops working on new tabs. However, I'm not sure if it only works on the first tab. 

I tried back with Vivaldi 6.0.2979.18, it is still working.
